core/src/main/scala/com/dimafeng/testcontainers/Container.scala

```diff
@@ -107,7 +108,7 @@ abstract class SingleContainer[T <: JavaGenericContainer[_]] extends TestContain
 
   def logConsumers: Seq[Consumer[OutputFrame]] = container.getLogConsumers.asScala.toSeq
 
-  def createContainerCmdModifiers: Set[Consumer[CreateContainerCmd]] = container.getCreateContainerCmdModifiers.asScala.toSet
+  def createContainerCmdModifiers: Set[CreateContainerCmdModifier] = container.getCreateContainerCmdModifiers.asScala.toSet
```

Comment:
@dimafeng This looks like a breaking change, but it's caused by the Java libraries' API change, I think we must follow it without choose?

Reply:
yes, I think so